CMakeLists.txt

```diff
@@ -47,7 +47,11 @@ ENDIF()
 
 IF(KHIVA_USE_CONAN)
     # Bring conan generated dependencies
-    INCLUDE(${CMAKE_CURRENT_BINARY_DIR}/conanbuildinfo.cmake)
+    if(EXISTS ${CMAKE_CURRENT_BINARY_DIR}/conanbuildinfo.cmake)
+        include(${CMAKE_CURRENT_BINARY_DIR}/conanbuildinfo.cmake)
+    else()
+        message(WARNING "The file conanbuildinfo.cmake doesn't exist, you have to run conan install first")
```

Comment:
Please, add a point at the end of the line and change "doesn't" by "does not". It should be "The file conanbuildinfo.cmake does not exist, you have to run conan install first."

Reply:
Please, could you find another way of showing that the conan install command should be used?  Something like "you have to run the "conan install" command first."